install

```diff
@@ -145,6 +145,11 @@ def chgrp?(path)
   !File.grpowned?(path)
 end
 
+# The USER env var is not guaranteed to be available, so fall back to id if not
+# provided. We destructively set it in the environment to make it available to
+# any sub-processes called.
+ENV["USER"] = `id -un`.chomp unless ENV["USER"]
```

Comment:
Use `||=` instead of `=` and then you can skip the `unless ...`

Reply:
Squash now @MikeMcQuaid? Or do you keep fixup/wip commits in your history?

Reply:
I prefer to squash.